lib/hawkular/hawkular_client.rb

```diff
@@ -47,13 +31,37 @@ def method_missing(name, *args, &block)
       delegate_client.__send__(method, *args, &block)
     end
 
+    def inventory
+      @inventory ||= Inventory::InventoryClient.create(entrypoint: "#{@state[:entrypoint]}/hawkular/inventory",
```

Comment:
I'm not sure, but at this point i don't want to break stuff like this: https://github.com/ManageIQ/manageiq/pull/13814/files#diff-7873cbd67d037fe54151ac4d277ccdd4R44